app/posts/manage-your-appointments/2024/10/2024-10-22-improving-nav.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Improving navigation within the MYA dashboard "
```

Comment:
Not sure you need to use service name/acronym in post title? Should be obvious from context? (And if not maybe we could make it more obvious somehow?)

```suggestion
title: "Improving navigation within the dashboard"
```